code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -1614,6 +1614,11 @@
 	icon_state = "cat"
 	color_src = HAIR
 
+/datum/sprite_accessory/tails/monkey/monky
```

Comment:
no fun allowed make this monkey/default

Reply:
@Fikou shame

Reply:
future gorillas, orangos, baboons  and chimpanzee place holders